client/src/components/css/AboutUs.css

```diff
@@ -9,27 +9,20 @@
     height: 50%;
     border: solid 1px #979797;
     background-color: #222222;
-    margin:auto;
+    /* margin:auto; */
     overflow:hidden;
     text-align:center;
     width: 100%;
     display: flex;
     justify-content: center;
     align-items: center;
+    padding: 10%;
+    padding-top: 5%;
+    padding-bottom: 5%;
 }
 
 .left {
-    float: left;
```

Comment:
Do the AboutUs css files mess with how the page looks?

Reply:
The AboutUs css files only mess w that card, but the image and text divs overlap if the page screen isn't wide enough.